buf.gen.yaml

```diff
@@ -7,5 +7,13 @@ plugins:
   - local: protoc-gen-connect-go
     out: gen
     opt: paths=source_relative
+  - local: protoc-gen-openapiv2
```

Comment:
what does this get over using the built-in json support

Reply:
i wanted to generate openapi swagger doc to be imported on tools like post man etc. to see the available endpoints but shall not touch the sdk generated, does connect generates swagger docs ??

Reply:
i think generating docs is a great idea. I found a package that looks like it does it for connect[protoc-gen-connect-openapi](https://github.com/sudorandom/protoc-gen-connect-openapi). Take a look at this draft pr as an example https://github.com/binarymatt/kayak/pull/5. I like that it generates yaml without any need to update proto files.

Reply:
i will pull some of your changes to the makefile in that pr as well